.docker/scripts/serve.sh

```diff
@@ -13,7 +13,7 @@ if [ "$VENV_DIR" != "" ]; then
   . ${VENV_DIR}/bin/activate
 fi
 if (which ckan > /dev/null); then
-    ckan -c ${CKAN_INI} run
+    ckan -c ${CKAN_INI} run -r
```

Comment:
what does -r do in this context?

Reply:
Disables the automatic reload of the server when files change. They shouldn't be changing anyway, but the build logs indicate that the reload is triggering.